### ISSUE TITLE
Display browser name in report

### ISSUE THREAD

**Issue by mistic100** [LironEr/cypress-mochawesome-reporter#180]
**Is your feature request related to a problem? Please describe.**

Hello, do you think it would make sense to display the browser on which the tests have ran somewhere in the report  ?

**Describe the solution you'd like**

Add a badge of the browser in the header after the title, it could use shields.io service

![](https://img.shields.io/badge/Firefox-grey.svg?logo=firefox)  
![](https://img.shields.io/badge/Chrome-grey.svg?logo=googlechrome)  
![](https://img.shields.io/badge/Edge-grey.svg?logo=microsoftedge)  
![](https://img.shields.io/badge/Electron-grey.svg?logo=electron)  

**Describe alternatives you've considered**

Append the name of the browser to `reportPageTitle` but I have no easy way to know which browser was requested when creating the configuration.

**Additional context**

**Comment by mistic100** [comment]
Turns out Mochawesome has zero options to customize the HTML output, you can't even add your own stylesheet !

Only solution I see is to append a custom script to modify the header after the initial render, kinda hacky...

**Comment by LironEr** [comment]
Yeah we cant customize the current HTML report without doing something hacky.

I started to work on a fork of mochawesome, because over the years I saw some more enhancements and the repo didnt have any activity for a long time.

Will update as soon as I have something to show
